### PR TITLE
Refactor fonts & images

### PR DESCRIPTION
**Pull request by razpeitia**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,13 +1,17 @@
 CC=gcc
-CFLAGS=-pipe -O2 -std=c11 -g -Wall -Wextra
+CFLAGS=-pipe -O2 -std=c11 -g -Wall -Wextra -I.
 LDFLAGS=$(shell pkg-config --cflags --libs sdl2 SDL2_image)
 
+DEPS = src/images.h src/fonts.h
+OBJ = src/netpong.o src/images.c src/fonts.c
```
Comment (razpeitia) [inline Makefile]: I have to check if `.o` or `.c`

File: src/fonts.c [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#include "fonts.h"
+
+const Uint32 FONT_HEIGHT = 16;
```
Comment (rdleon) [inline src/fonts.c]: We should agreed on when to use `#define` and when to use `const`. For this type of things I prefer `#define`

File: src/netpong.c [hunk 3]
```diff
@@ -456,11 +303,11 @@ int main()
 
         for(int i = 0; i < 100; i++) buffer[i] = 0;
```
Comment (rdleon) [inline src/netpong.c]: Missed in a past review. Can be changed for `memset(buffer, 0, MAX_TEXT_BUFF_SIZE);` or something similar

File: src/fonts.c [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+void load_font(SDL_Renderer* renderer) {
+    Font.texture = load_image(renderer, "images/good_neighbors.png");
```
Comment (alimon) [inline src/fonts.c]: Create argument for  "images/good_neighbors.png" as const char *

File: Makefile (hunk 1 above)
Comment (alimon) [reply Makefile]: Yes is preferred .o.

File: src/fonts.c (hunk 2 above)
Comment (alimon) [reply src/fonts.c]: +1

File: src/fonts.c [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    for(Uint32 i = 0; i < 128; i++) {
+        Font.rects[i].x = 0;
+        Font.rects[i].y = 0;
+        Font.rects[i].w = 0;
+        Font.rects[i].h = FONT_HEIGHT;
+    }
+    FILE *f = fopen("data/font.txt", "r");
```
Comment (alimon) [inline src/fonts.c]: Create argument for "data/font.txt" as const char *

File: Makefile [hunk 6]
```diff
@@ -1,13 +1,17 @@
 CC=gcc
-CFLAGS=-pipe -O2 -std=c11 -g -Wall -Wextra
+CFLAGS=-pipe -O2 -std=c11 -g -Wall -Wextra -I.
```
Comment (rdleon) [inline Makefile]: I would add the `-I` to LDFLAGS

File: src/fonts.c (hunk 2 above)
Comment (nullset2) [reply src/fonts.c]: Care to explain why do you prefer the preprocessor here? I tend to not like it as much.

**Pull request merged**